src/config/ReportConfig.tsx

```diff
@@ -459,123 +450,6 @@ export const REPORT_TYPES = {
                 type: SELECTION_TYPES.NUMBER,
                 default: 40
             },
-            "autorun": {
-                label: "Auto-run query",
-                type: SELECTION_TYPES.LIST,
-                values: [true, false],
-                default: true
-            },
-            "description": {
-                label: "Report Description",
-                type: SELECTION_TYPES.MULTILINE_TEXT,
-                default: "Enter markdown here..."
-            }
-        }
-    },
-    "radar": {
```

Comment:
same here, a lot of things have vanished in your branch. Did you rebase with the latest develop?